ctrd/client.go

```diff
@@ -166,3 +185,125 @@ func (c *Client) Version(ctx context.Context) (containerd.Version, error) {
 
 	return cli.client.Version(ctx)
 }
+
+func (c *Client) runContainerdDaemon(homeDir string, copts clientOpts) error {
+	if homeDir == "" {
```

Comment:
Could we use `os.Stat` to check homeDir here? If the homeDir doesn't exist, we should return error right away.
WDYT?

Reply:
the home-dir existence is checked in main.go, so we no need check here.
but we should check the `stateDir` here, thanks for your reminder, :)